test/test-SchemaTest.cc

```diff
@@ -0,0 +1,117 @@
+//
+//  test-RenderTest.cc
+//  drafter
+//
+//  Created by Vilibald Wanča on 19/11/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+
+#include "draftertest.h"
+
+using namespace draftertest;
+
+TEST_REFRACT("schema", "boolean");
+TEST_REFRACT("schema", "number");
+TEST_REFRACT("schema", "string");
+TEST_REFRACT("schema", "required");
+TEST_REFRACT("schema", "optional");
+TEST_REFRACT("schema", "object-simple");
+TEST_REFRACT("schema", "required-object");
+TEST_REFRACT("schema", "array-simple");
+TEST_REFRACT("schema", "description");
+TEST_REFRACT("schema", "required-array");
+TEST_REFRACT("schema", "default-sample");
+// FIXME:  enum bug #176
+TEST_REFRACT("schema", "default-attribute");
+TEST_REFRACT("schema", "escaping");
+TEST_REFRACT("schema", "array-inline");
+TEST_REFRACT("schema", "array-of-arrays");
+TEST_REFRACT("schema", "array-of-types");
+TEST_REFRACT("schema", "array-of-types-only");
+TEST_REFRACT("schema", "array-with-nested-type");
+TEST_REFRACT("schema", "array-with-nested-types");
+TEST_REFRACT("schema", "array-of-types-mixed");
+TEST_REFRACT("schema", "array-of-types-mixed-complex");
```

Comment:
The above 7 test cases are wrong. Boutique has them wrong and drafter was supposed to fix them.